src/monitor.rs

```diff
@@ -52,17 +52,41 @@ impl Iterator for AvailableMonitorsIter {
 /// - [`MonitorHandle::video_modes`][monitor_get].
 ///
 /// [monitor_get]: ../monitor/struct.MonitorHandle.html#method.video_modes
-#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+#[derive(Derivative)]
+#[derivative(Clone, Debug = "transparent", PartialEq, Eq, Hash)]
 pub struct VideoMode {
-    pub(crate) size: (u32, u32),
-    pub(crate) bit_depth: u16,
-    pub(crate) refresh_rate: u16,
+    pub(crate) video_mode: platform_impl::VideoMode,
+}
+
+impl PartialOrd for VideoMode {
+    fn partial_cmp(&self, other: &VideoMode) -> Option<std::cmp::Ordering> {
+        Some(self.cmp(other))
+    }
+}
+
+impl Ord for VideoMode {
+    fn cmp(&self, other: &VideoMode) -> std::cmp::Ordering {
+        // TODO: we can impl `Ord` for `PhysicalSize` once we switch from `f32`
+        // to `u32` there
+        let size: (u32, u32) = self.size().into();
```

Comment:
What is this code sorting by? Can we at least get a comment for the users?

Reply:
Looks like it's sorting by Monitor->Size->Refresh Rate, which seems like the natural way to sort video modes (thinking in terms of how you'd sort video modes in a settings list). I personally don't feel too strongly about requiring docs here, since this is the most intuitive way to sort the video modes anyway, but I'm inclined leave that detail up to @aleksijuvani.

Reply:
Ah, that makes sense. Was tired this morning, spent five minutes staring at it and couldn't figure out what it did. Yeah, I don't think this needs any docs.